test/ci/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN apt-get install -yq git gcc gdc
 RUN apt-get install -yq --no-install-recommends \
     cmake \
     intltool \
+    libstemmer-dev \
```

Comment:
Not needed anymore, I updated the dependencies in master (and dropped most of them).